File: backend/src/main/java/com/coolparty/restaurant/model/pojo/Reservation.java
package com.coolparty.restaurant.model.pojo;

import lombok.Data;

import java.time.LocalDateTime;
import java.util.UUID;

@Data
public class Reservation extends IdentifiableNamedEntity<UUID> {
    private UUID tableId;
    private String email;
    private LocalDateTime reservedFrom;
    private LocalDateTime reservedTo;
}

File: frontend/www/src/app/modules/core/_model/Reservation.ts
export class Reservation {

  id: string;

  name: string;

  tableId: string;

  email: string;

  reservedFrom: Date;

  reservedTo: Date;

  constructor(id, name, tableId, email, reservedFrom, reservedTo) {
    this.id = id;
    this.name = name;
    this.tableId = tableId;
    this.email = email;
    this.reservedFrom = reservedFrom;
    this.reservedTo = reservedTo;
  }
}
